### PR TITLE
Add docker.ssh.privateKeyFile property.

### PR DESCRIPTION
**Pull request by dudaerich**

### CODE REVIEW

**Comment by kwart** [comment]
Please add `CHANGELOG.md` entry too.

File: docker-README.md [hunk 1]
```diff
@@ -100,25 +100,26 @@ has to be provided in  `node.[name].ssh.*` properties.
...
+| docker.portBindings       | Comma separated list of TCP port bindings in form "hostPort:nodePort" |                                |
+| docker.privileged         | true/false flag which controls if the container is privileged     | false                              |
+| docker.ssh.port           | SSH port number if SSH is installed (not-mapped - i.e. in-container value) |                           |
+| docker.ssh.user           | SSH username                                                      |                                    |
+| docker.ssh.password       | SSH user password                                                 |                                    |
+| docker.ssh.privateKey     | The SSH private key (not the file of the key!). In case you want to pass a private key manually. Currently unused in favor of `docker.ssh.privateKeyFile`. | [None. Optional.] |
+| docker.ssh.privateKeyFile | The path to the private key which should be used to connect to the instance. Necessary for establishing an SSH channel in your tests. | [None. Optional.] |
```
Comment (LittleJohnII) [inline docker-README.md]: I think we should either document that `docker.ssh.privateKeyFile` overrides `docker.ssh.privateKey` or check whether both are supplied, and if yes, fail.

Comment (kwart) [inline docker-README.md]: Please reuse descriptions from `openstack-README.md`.

**Comment by dudaerich** [comment]
Thanks for your comments. I updated the PR, please review it again.

**Comment by LittleJohnII** [comment]
Looks good to me.

**Comment by kwart** [comment]
LGTM. Merging. Thanks for the feature.

**Pull request merged**